packages/card/Card.tsx

```diff
@@ -1,28 +1,25 @@
 import { h, Component } from 'skatejs';
 import styles from './Card.scss';
+import { shadyCssStyles } from '../_helpers';
```

Comment:
please rebase and update it to use common package as Martin did manage to merge it on Friday.

Thanks